src/services/sms.py

```
@@ -0,0 +1,62 @@
+    }
+
+    async with httpx.AsyncClient(timeout=httpx.Timeout(5.0, connect=2.0)) as client:
+        res = await client.post(
+            url,
+            json=payload,
+            headers=headers,
+        )
+    if res.status_code >= 400:
+        raise RuntimeError(
+            f"sms provider request failed with status={res.status_code}, body={res.text}"
+        )
```

Ruff (0.15.2)

[warning] Avoid specifying long messages outside the exception class

(TRY003)